build.gradle

```diff
@@ -150,6 +150,10 @@ integTest {
     // allows integration test classes to access test resource from project root path
     systemProperty('project.root', project.rootDir.absolutePath)
 
+    systemProperty "https", System.getProperty("https")
+    systemProperty "user", System.getProperty("user")
```

Comment:
Will these properties change how we run "./gradlew build"?

Reply:
No they do not have any impact.  These will be used only when ```-Dhttps=true``` is passed along with ./gradlew build